snippets/saved_replies.md

```diff
@@ -0,0 +1,105 @@
+This document is useful for copy and pasting into https://github.com/settings/replies - feel free to change wording to your preference.
+It's good to have a bit of variety in our responses, anyway.
+If should be possible to combine any of these snippets. Simply cut off the initial thank you blurb from a snippet and replace with "Also,". Any combination should create a sane flow, even if most of them obviously wouldn't make much sense.
+
+All these snippets follow the same basic layout
+
+1. Thank people.
+They actually took the time to create an issue/PR. Some took more time, some less time; for some it was hard, for others it was trivial. Disregarding all this, we simply thank the first as they are doing more than 99% or our userbase.
+2. Acknowledge their need for help.
+Unless it's a PR for a typo, most likely they actually feel a need for whatever they are asking about.
+2. Give context.
+Considerations, better places to go with this, etc. If it's anything we have docs on, enable them to come up to speed with the factual basis we have.
+3. Provide a course of action.
+At the least, if we close anything quickly, we should tell them they can ask to re-open, or where else to take this. Again, take into account that there's a need they feel they need to address, somehow.
+
+It's not always feasible, or even possible, to fill out the XXX. Still, we should try to.
```

Comment:
I was confused and didn't understand that the XXX here wasn't a placeholder itself, but referred to subsequent placeholders. How about `the "XXX" placeholders below`?